src/com/github/kornilova203/matlab/completion/MatlabKeywordCompletionContributor.kt

```diff
@@ -18,6 +18,14 @@ class MatlabKeywordCompletionContributor : CompletionContributor() {
         private val IDENT = psiElement(MatlabTypes.IDENTIFIER).withParent(MatlabRefExpr::class.java)
         private val AT_TOP_LEVEL = and(M, IDENT.withSuperParent(2, MatlabFile::class.java))
         private val IN_BLOCK = and(M, IDENT.withSuperParent(2, MatlabBlock::class.java))
+        private val AFTER_INTEGER_LITERAL = psiElement().afterLeafSkipping(
+                psiElement().withText(""),
+                psiElement().withElementType(MatlabTypes.INTEGER)
+        )
+        private val AFTER_FLOAT_LITERAL = psiElement().afterLeafSkipping(
+                psiElement().withText(""),
+                psiElement().withElementType(MatlabTypes.FLOAT)
```

Comment:
There is no need to create two different rules. You may create composite rule like this:
```kotlin
        private val AFTER_NUMBER_LITERAL = psiElement().afterLeafSkipping(
                alwaysFalse<PsiElement>(),
                or(psiElement(MatlabTypes.INTEGER), psiElement(MatlabTypes.FLOAT))
        )
```